### PR TITLE
User can add attachments to an assignment

### PR DESCRIPTION
**Pull request by margaretoti**
[Trello card](https://trello.com/c/jv1d2jDD). This branch allows users to add multiple attachments to an assignment when creating an assignment. The attachment of files is handled using the [Paperclip gem](https://github.com/thoughtbot/paperclip). Currently users can only add files; they cannot edit or delete them. 

screen shot 2017-05-25 at 11 55 05 am

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 /log/*.log
 /tmp
 tmp
+/public
```
Comment (margaretoti) [inline .gitignore]: The test suite stored all files in this hugely nested folder.

File: Gemfile [hunk 2]
```diff
@@ -13,6 +13,7 @@ gem "jquery-rails"
 gem "neat", "~> 1.7.0"
 gem "normalize-rails"
 gem "paper_trail"
+gem "paperclip", "~> 5.0.0"
```
Comment (margaretoti) [inline Gemfile]: @derekprior, I could not remember which version of Paperclip you updated to so I just copied the one in the Paperclip documentation. Happy to `bundle install` a different version.

File: .gitignore (hunk 1 above)
Comment (derekprior) [inline .gitignore]: Some things in public we actually want. For instance, the error pages are stored in there. I think `public/system` should be good.

File: Gemfile (hunk 2 above)
Comment (derekprior) [inline Gemfile]: This is good.

**Comment by margaretoti** [comment]
@derekprior, Sounds good. I will wait for you to finish adding attachments to coverage. In the meantime, I might try to write a model validation to check that a user attaches only approved file formats (pdf for now).

**Pull request merged**